### PR TITLE
AbstractShutter remove specific states

### PR DESCRIPTION
**Pull request by beteva**
Remove ShutterStates. Upgrade ShutterMockup to use AbstractShutter.

### CODE REVIEW

File: HardwareObjects/mockup/ShutterMockup.py [hunk 1]
```diff
@@ -17,28 +17,46 @@
...
     ShutterMockup for simulating a simple open/close shutter.
     """
 
+    SPECIFIC_STATES = ShutterStates
+
     def init(self):
+        """Initialisation"""
         super(ShutterMockup, self).init()
-        self.update_value(self.VALUES.CLOSED)
+        self.update_value(self.VALUES.CLOSE)
```
Comment (rhfogh) [inline HardwareObjects/mockup/ShutterMockup.py:49]: @beteva You are *sure* that this should be 'CLOSE', and not 'CLOSED'??
'CLOSE' is either a verb ('fermer') which is strange for a value (not an action), or it is an adjective meaning 'not far away ('proche'). 'CLOSED' would be the obvious linguistical choice

Comment (beteva) [reply HardwareObjects/mockup/ShutterMockup.py:49]: @rhfogh, I was thinking as a verb, but you are right, it should be a noun. I will change it.

**Comment by IvarsKarpics** [comment]
Are we happy with this PR? I am fine with it.

**Comment by marcus-oscarsson** [comment]
Agree :)

**Comment by beteva** [comment]
Done 5 days ago.

**Pull request merged**